### DISCUSSION TITLE
MOCO 2023 - Ideenfindung

### DISCUSSION THREAD

**Discussion by mjung2605** [mjung2605/MOCO2023#1] [Ideas]
Auszug aus den Folien:

> Feel free to add some startup flavour ;-)
Some requirements
-Your idea needs to be characteristic for the mobile domain
-Your idea must make reasonable use of mobile features (Like _**sensors, notifications, persistence, async events, multi-threading, work manager, cloud backend**_ etc.); use appropriate _**frameworks**_ and _**APIs**_ correctly
Project idea needs to be approved by supervisors during first project milestone <

Ideensammlung:
- TODO-App mit Notifications (und evtl der Möglichkeit, Fotos zu adden)
- App, die GPS des Handys nutzt, um gewisse Orte in der Nähe anzuzeigen (welche?/Was ist der Fokus dieser App?) (Achtung: eine ähnliche Idee hat eine andere Gruppe mit Sehenswürdigkeiten)
- App, die GPS des Handys nutzt, um z.B. körperliche Aktivität zu messen (z.B. Schritte) -> Einbettung in eine Art "Spiel" (-> Motivation für Sport ist gut ;) )
- Irgendein Spiel, welches die Bewegungssensoren benutzt (Balance?)
- ...
- etc

**Comment by sebfuh** [comment]
Ideensammlung:

 - Art "gebündelte" Informationsquelle/App für die TH Köln (Standort GM), mit Inhalt - 
     - Stundenplan für versch. Studiengänge 
     - Schwarzes Brett
     - Gebäudeplan?/ Raumplan?
     -  aktuelle Informationen
     - Klausurplan
     - Mensaplan
     - sonstige für Studenten nützliche Sachen

**Comment by angi-mrer** [comment]
- Personalisierbarer Fahrplan 
- Nachhilfe App - verschiedene Räume? 
- TH Fundsachen App (https://www.ratio-software.com/produkte/web-services-und-internet-tools/ratioww-fundsachen-app/) 
- Klausurenplaner
- Open AI?
- Study App - Kalender, Suchfunktion, Verbindung zu Youtube -> Livestream Study Music

**Reply by mjung2605** [reply]
Fundsachen: evtl. auf Map darstellen

**Reply by sebfuh** [reply]
https://developer.trademe.co.nz/api-reference/my-trade-me-methods/retrieve-your-lost-items evtl. passende Schnittstellen-API
